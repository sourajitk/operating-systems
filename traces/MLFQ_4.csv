1,0,15
2,4,4
3,5,2
4,9,4
5,11,3
6,14,5