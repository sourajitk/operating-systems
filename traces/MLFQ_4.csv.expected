1,33
2,19
3,8
4,20
5,17
6,23